### ISSUE TITLE
Wrong russian translations

### ISSUE THREAD

**Issue by kotmatpockuh** [froala/wysiwyg-editor#1123]
http://prntscr.com/af3eq3 - Полужирный
http://prntscr.com/af3eyb - Выровнять по
http://prntscr.com/af3f3h - **Т**екст
http://prntscr.com/af3feg - Добавить URL
http://prntscr.com/af3fmp - Переместите сюда файл
http://prntscr.com/af3fxx - Обновить
http://prntscr.com/af3g4i - Заменить

There a lot of other wrong translations too. But above are the most critical...
Also: every translation must be from upper-case or from lower-case. Mixing them is a bad idea.

**Comment by dianaprajescu** [comment]
Unfortunately, we have no Russian speaker among us. The only way to improve the translation is with help from our users. I just made the changes you suggested. If there is any other change you consider necessary, please let us know.

**Comment by kotmatpockuh** [comment]
@dianaprajescu 

Hi. Are you sure, that this lang-params are in released v2.2.3?

**Comment by dianaprajescu** [comment]
I just checked and the corrections you gave us are there. For example Bold: 
<img width="204" alt="screen shot 2016-04-11 at 15 17 36" src="https://cloud.githubusercontent.com/assets/1551136/14426574/9da740ea-fff8-11e5-8659-e2a45a2f3eb6.png">

Please make sure that you have updated the /languages/ru.js file when upgrading to the new version. If you are using Bower, the language files are not included in Bower, they need to be included separately.

**Comment by kotmatpockuh** [comment]
i'm using angular-froala, and version... 2.2.3 - but language files are still old

**Comment by kotmatpockuh** [comment]
@dianaprajescu
it's simple...
you didn't update this one: http://prntscr.com/ar81e3 
"http://prntscr.com/af3fmp - Переместите сюда файл"

**Comment by kotmatpockuh** [comment]
please, reopen this bug!

**Comment by dianaprajescu** [comment]
Yes, it seems I missed that one. Just made the update.

I would really appreciate if next time you have translation suggestions, just say "Drop file: Переместите сюда файл" or add the image here, directly on Github, instead of adding multiple links to different images, which makes them very hard to follow for a person that doesn't understand Russian.
